src/rng/backends/rocrand/CMakeLists.txt

```diff
@@ -1,103 +1,91 @@
-# distribute, and sublicense such enhancements or derivative works thereof,
-# in binary and source code form.
+# You are under no obligation whatsoever to provide any bug fixes, patches, or
+# upgrades to the features, functionality or performance of the source code
+# ("Enhancements") to anyone; however, if you choose to make your Enhancements
+# available either publicly, or directly to Lawrence Berkeley National
+# Laboratory, without imposing a separate written license agreement for such
+# Enhancements, then you hereby grant the following license: a non-exclusive,
+# royalty-free perpetual license to install, use, modify, prepare derivative
+# works, incorporate into other computer software, distribute, and sublicense
+# such enhancements or derivative works thereof, in binary and source code form.
 #
 # If you have questions about your rights to use or distribute this software,
-# please contact Berkeley Lab's Intellectual Property Office at
-# IPO@lbl.gov.
+# please contact Berkeley Lab's Intellectual Property Office at IPO@lbl.gov.
 #
-# NOTICE.  This Software was developed under funding from the U.S. Department
-# of Energy and the U.S. Government consequently retains certain rights.  As
-# such, the U.S. Government has been granted for itself and others acting on
-# its behalf a paid-up, nonexclusive, irrevocable, worldwide license in the
-# Software to reproduce, distribute copies to the public, prepare derivative
-# works, and perform publicly and display publicly, and to permit others to do
-# so.
-#=================================================================================
+# NOTICE.  This Software was developed under funding from the U.S. Department of
+# Energy and the U.S. Government consequently retains certain rights.  As such,
+# the U.S. Government has been granted for itself and others acting on its
+# behalf a paid-up, nonexclusive, irrevocable, worldwide license in the Software
+# to reproduce, distribute copies to the public, prepare derivative works, and
+# perform publicly and display publicly, and to permit others to do so.
+# =================================================================================
 
 set(LIB_NAME onemkl_rng_rocrand)
 set(LIB_OBJ ${LIB_NAME}_obj)
 find_package(rocRAND REQUIRED)
 
-set(SOURCES philox4x32x10.cpp
-  mrg32k3a.cpp
-  $<$<BOOL:${BUILD_SHARED_LIBS}>: mkl_rng_rocrand_wrappers.cpp>)
+set(SOURCES philox4x32x10.cpp mrg32k3a.cpp $<$<BOOL:${BUILD_SHARED_LIBS}>:
+            mkl_rng_rocrand_wrappers.cpp>)
 
 add_library(${LIB_NAME})
 add_library(${LIB_OBJ} OBJECT ${SOURCES})
 
-target_include_directories(${LIB_OBJ}
-  PRIVATE ${PROJECT_SOURCE_DIR}/include
-          ${PROJECT_SOURCE_DIR}/src
-          ${CMAKE_BINARY_DIR}/bin
-          ${MKL_INCLUDE}
-)
+target_include_directories(
+  ${LIB_OBJ} PRIVATE ${PROJECT_SOURCE_DIR}/include ${PROJECT_SOURCE_DIR}/src
+                     ${CMAKE_BINARY_DIR}/bin ${MKL_INCLUDE})
```

Comment:
There are many whitespace changes in CMake files including copyright and cmake code. Can you please share why we are making those changes?

Reply:
We've observed that the format of this CMake file was not proper when compared with the other CMake files.
Hence we've ran the (cmake-format)[https://cmake-format.readthedocs.io/en/latest/] tool on this file.

If you have any suggestion or recommendation on handling CMake file formats for oneMKL, do share with us.

Reply:
I see, I think it is a great idea. @mkrainiuk do you think it would help us to investigate this tool and add it to Contribution guidelines just like clang-format?

Reply:
Yes, this is a good idea to improve CMake file format too.